test/e2e/filters/http_body/main.go

```diff
@@ -5,26 +5,41 @@ import (
 )
 
 func init() {
-	gonvoy.RunHttpFilter(new(BodyReadFilter), gonvoy.ConfigOptions{
-		FilterConfig: new(Config),
+	gonvoy.RunHttpFilter(filterName,
```

Comment:
similarly on this one, `filterName` must differ on every filters being registered